client.js

```diff
@@ -142,6 +143,9 @@ class Client {
     });
   }
 
+  /**
+   * Request the list of logged nodes.
```

Comment:
If you are commenting the API, should try say something more that just repeat the method name if possible. E.g. here: "Request the list of logged nodes. In CDP Studio, this matches the LoggedValues table of the CDPLogger component."